docs/data/old-version/dea-fractional-cover-landsat-2.3.1/_data.yaml

```diff
@@ -0,0 +1,86 @@
+# See the Product metadata fields documentation:
+# https://docs.dev.dea.ga.gov.au/public_services/dea_knowledge_hub/product_metadata_fields.html
+
+# Overview
+
+title: DEA Fractional Cover (Landsat)
+long_title: Geoscience Australia Landsat Fractional Cover Collection 3
+header_image: /_files/cmi/FractionalCover1.jpg
+version: 2.3.1
+is_latest_version: false
+latest_version_link: /data/product/dea-fractional-cover-landsat/
+product_type: Derivative
+spatial_data_type: Raster
+time_span:
+  start: 16/08/1986
```

Comment:
ISO format:
```suggestion
  start: 1986-08-16
```

Reply:
ISO format is good and scientists use it a lot. There is another format that is also quite a good one. Do you have a preference:

`1986-08-16`

vs

`16 Aug 1986`

Reply:
I prefer 1986-08-16 :)

Reply:
Hi @tebadi, I conducted a project where I investigated what is the date format with the aim of standardising them across the Knowledge Hub. https://github.com/GeoscienceAustralia/dea-knowledge-hub/pull/273 

> I asked several stakeholders for their opinions on these date formats and I also consulted the [Australian Government Style Manual](https://www.stylemanual.gov.au/grammar-punctuation-and-conventions/numbers-and-measurements/dates-and-time#combine_numerals_and_words_for_dates_in_body_text). Therefore, I concluded that the appropriate date format is 1 Jan 2024. This aligns with the Style Manual which encourages the use of a worded Month name, due to the ambiguities of numbered date names. Also, the worded Month name may allow the date to be easier to read for some readers.

I'm wondering if it's ok if I change the date format to `1 Jan 2024` style?

Reply:
Sure, go ahead 👍